From d1d82d923c65daa4983b91d4c3bde43ff4271c7a Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Thu, 11 Jun 2020 00:50:49 -0700
Subject: [PATCH 1169/1239] Revert "drivers: watchdog: Enable for OcteonTX3
 platforms"

This reverts commit 02803d2d3d8e4264a027d8eb366f9deaef17beea.

Reason for revert: Incorrect platform name

Change-Id: I89467a69ab2b9b81fd9d7651471abd15af371ea4
Signed-off-by: Suneel Garapati <sgarapati@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/boot/u-boot/+/29833
Reviewed-by: Suneel Garapati <sgarapati@caviumnetworks.com>
Tested-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 drivers/watchdog/Kconfig        | 4 ++--
 drivers/watchdog/octeontx_wdt.c | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/watchdog/Kconfig b/drivers/watchdog/Kconfig
index 9fb080c4b6..68bfa92f41 100644
--- a/drivers/watchdog/Kconfig
+++ b/drivers/watchdog/Kconfig
@@ -114,8 +114,8 @@ config WDT_MTK
 
 config WDT_OCTEONTX
 	bool "OcteonTX core watchdog support"
-	depends on WDT && (ARCH_OCTEONTX || ARCH_OCTEONTX2 || ARCH_OCTEONTX3)
-	default y if WDT && ARCH_OCTEONTX || ARCH_OCTEONTX2 || ARCH_OCTEONTX3
+	depends on WDT && (ARCH_OCTEONTX || ARCH_OCTEONTX2)
+	default y if WDT && ARCH_OCTEONTX || ARCH_OCTEONTX2
 	imply WATCHDOG
 	help
 	  This enables OcteonTX watchdog driver, which can be
diff --git a/drivers/watchdog/octeontx_wdt.c b/drivers/watchdog/octeontx_wdt.c
index e558768422..9362ad99bf 100644
--- a/drivers/watchdog/octeontx_wdt.c
+++ b/drivers/watchdog/octeontx_wdt.c
@@ -16,7 +16,7 @@ DECLARE_GLOBAL_DATA_PTR;
 
 #if CONFIG_IS_ENABLED(ARCH_OCTEONTX)
 #define REG_BASE 0x844000000000
-#else
+#elif CONFIG_IS_ENABLED(ARCH_OCTEONTX2)
 #define REG_BASE 0x802000000000
 #endif
 
-- 
2.29.0

